python3-poetry-core.yaml

```diff
@@ -0,0 +1,10 @@
+# Generated with flatpak-pip-generator --yaml poetry-core==1.9.0
+name: python3-poetry-core
+buildsystem: simple
+build-commands:
```

Comment:
this is a build-only dependency ? If so it should be cleaned up using cleanup with `- "*"`

Reply:
yes, and done